src/utils/update/company.profile.mapper.to.oversea.entity.ts

```diff
@@ -4,7 +4,6 @@ import { Entity } from "../../model/entity.model";
 
 export const mapCompanyProfileToOverseasEntity = (cp: CompanyProfile): Entity => {
   return {
-    registration_number: cp.companyNumber,
```

Comment:
This needs to be fetched from CompanyProfile.foreignCompanyDetails
        It may be missing and needs to be added in the node-sdk

Reply:
Now have a PR for sdk: https://github.com/companieshouse/api-sdk-node/pull/531